docs/reference/data-management/migrate-index-allocation-filters.asciidoc

```diff
@@ -24,7 +24,7 @@ indices to switch to node roles.
 [[cloud-migrate-to-node-roles]]
 === Automatically migrate to node roles on {ess} or {ece}
 
-If you are using node attributes on {ess} or {ece}, you will be
+If you are using node attributes from the default template {ess} or {ece}, you will be
```

Comment:
@stefnestor , what `template` is this?  Is this an index template from Filebeat, or a deployment template from the orchestration system, or something else?